### PR TITLE
ci: fixes to address upcoming ibis incompatibilities

### PR DESCRIPTION
**Pull request by cpcloud**
This PR is a test of a PR to address the failing ibis master builds here

### CODE REVIEW

**Comment by cpcloud** [comment]
This won't pass until https://github.com/ibis-project/ibis/pull/3240 is merged.

Luckily there's nothing here that requires a major release.

File: ci/install_deps.sh [hunk 1]
```diff
@@ -13,10 +13,10 @@ import sys
...
 $(cat ./ibis/setup.py)" > ./ibis/setup.py
 
-    python -m pip install --user -e ./ibis
+    python -m pip install --user ./ibis
 else
     python -m pip install --user ibis-framework=="$IBIS_VERSION"
 fi
 
-python -m pip install --user -e .
+python -m pip install --user ./ibis_bigquery
```
Comment (cpcloud) [inline ci/install_deps.sh:21]: The nested install (checking out `ibis` inside of `ibis_bigquery`) was causing duplicated results for the `ibis.backends.bigquery` entry point from `importlib.metadata.entry_points()`.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -62,7 +62,9 @@ jobs:
     steps:
 
     - name: checkout
-      uses: actions/checkout@v1
+      uses: actions/checkout@v2
+      with:
+        path: ibis_bigquery
```
Comment (tswast) [inline .github/workflows/main.yml:67]: Need this in https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/system-tests-pr.yml and https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/system-tests.yml as well, I believe.

**Review by tswast — APPROVED** [review]
I recall that `pull_request_target` uses the config from `main` branch, so I think we'll need to merge this before we know if it actually works.

**Comment by tswast** [comment]
Oops. Looks like it's failing due to some auth issues: https://github.com/ibis-project/ibis-bigquery/runs/4716610653?check_suite_focus=true

Might need to update https://github.com/ibis-project/ibis-bigquery/blob/main/ci/decrypt_secret.sh or more likely https://github.com/ibis-project/ibis-bigquery/blob/main/.github/workflows/system-tests-pr.yml#L58

**Comment by cpcloud** [comment]
I'll put up a change to revert the environment variable changes.

**Comment by tswast** [comment]
https://github.com/ibis-project/ibis-bigquery/pull/108 might fix it. I think it has more to do that we're extracting the encrypted secret into the same directory, so we need to update the path for the new location.

**Comment by tswast** [comment]
Actually maybe not. I see `decrypt_secrets.sh` uses an absolute path. Reverting those lines should work.

**Pull request merged**